glib/src/param_spec.rs

```diff
@@ -205,6 +211,36 @@ impl ParamSpec {
     //pub fn steal_qdata(&self, quark: /*Ignored*/glib::Quark) -> /*Unimplemented*/Option<Fundamental: Pointer> {
     //    unsafe { TODO: call gobject_ffi::g_param_spec_steal_qdata() }
     //}
+
+    #[doc(alias = "g_param_spec_internal")]
+    pub fn internal(
```

Comment:
I think exposing it like this is not very useful. You'd have to provide some more infrastructure like for `GObject`s, and also expose `g_param_type_register_static()` / `GParamSpecTypeInfo` in some way (by providing some trait, I guess).

Reply:
Right, I don't plan on doing so as I don't really need it. I guess we can keep it and whoever needs that can provide that infra or drop the commit, whatever you prefer

Reply:
I would drop it as whatever we're going to expose here will look different